### ISSUE TITLE
Incompatibility with Puppet 3.8

### ISSUE THREAD

**Issue by ghost** [theforeman/puppet-foreman_scap_client#34]
Puppet 3.8 is included in Red Hat Satellite. This plugin isn't compatible with this version. In puppet 3.8 the "else" statement isn't allowed after "unless". Please switch to "if".

```
# puppet --version
3.8.6
# puppet parser validate *pp
Error: Could not parse for environment production: Syntax error at 'else'; expected '}' at manifests/init.pp:73
```

**Comment by ares** [comment]
Thank you @scitechfh, would you mind testing the fix before I merge? Please see https://github.com/theforeman/puppet-foreman_scap_client/pull/35

**Comment by ghost** [comment]
Thank you for fixing. I have no parsing error with puppet 3.8

**Comment by ares** [comment]
Thanks, merged and released in 0.3.15, package should appear in foreman repositories during today.
